### PR TITLE
Doxygen Configuration File

### PR DESCRIPTION
**Pull request by marip8**
This PR adds the doxygen configuration file from the [industrial_moveit](https://github.com/ros-industrial/industrial_moveit) project per #8 so that documentation for STOMP can be generated

### CODE REVIEW

File: doxygen.config [hunk 1]
```diff
@@ -0,0 +1,2303 @@
...
+
+DOXYFILE_ENCODING      = UTF-8
+
+# The PROJECT_NAME tag is a single word (or a sequence of words surrounded by
+# double-quotes, unless you are using Doxywizard) that should identify the
+# project for which the documentation is generated. This name is used in the
+# title of most generated pages and in a few other places.
+# The default value is: My Project.
+
+PROJECT_NAME           = "Industrial Moveit"
```
Comment (gavanderhoorn) [inline doxygen.config]: Should this be changed to reflect the new location?

File: doxygen.config [hunk 2]
```diff
@@ -0,0 +1,2303 @@
...
+# Configuration options related to the input files
+#---------------------------------------------------------------------------
+
+# The INPUT tag is used to specify the files and/or directories that contain
+# documented source files. You may enter file names like myfile.cpp or
+# directories like /usr/src/myproject. Separate the files or directories with
+# spaces.
+# Note: If this tag is empty the current directory is searched.
+
+INPUT                  = constrained_ik stomp_core stomp_moveit stomp_plugins
```
Comment (gavanderhoorn) [inline doxygen.config]:
```suggestion
INPUT                  = stomp_core stomp_moveit stomp_plugins
```

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
As this was a direct copy a few things need updating I believe.

**Review by jrgnicho — APPROVED** [review]

**Comment by jrgnicho** [comment]
fixes #8

**Pull request merged**